src/controller/clients/shape-search-client.js

```diff
@@ -52,11 +52,13 @@ class ShapeSearchClient extends BaseClient {
     for (let unit in units) {
       // console.log("Unit: ", units[unit])
       for (let i in mObj) { mObj[i] = ""  }
-      let mObjPush = JSON.parse(JSON.stringify(mObj))
-      mObjPush.location = units[unit][0].location
-      mObjPush.condo = true
-      mObjPush.pwd_parcel_id = units[unit][0].pwd_parcel_id
-      data.rows.push(mObjPush)
+      let mObjPush = JSON.parse(JSON.stringify(mObj));
+      mObjPush.owner_1 = "Condominium (" + units[unit].length + " Units)";
+      mObjPush.owner_2 = null;
```

Comment:
Is this line necessary?

Reply:
Yes. There is another function elsewhere that joins these two fields with a comma, so the blank space triggers that part of the if statement.